client/me/help/help-search/index.jsx

```diff
@@ -77,6 +78,9 @@ module.exports = React.createClass( {
 			);
 		}
 
+		const supportLocale = olarkStore.get().locale;
```

Comment:
It seems weird that we're getting this from Olark. Is there a more general place we can get it from? Eventually Olark will go away, so I'm not sure having this dependency in place is good.